src/code42cli/cmds/legal_hold.py

```diff
@@ -124,6 +138,37 @@ def show(state, matter_id, include_inactive=False, include_policy=False):
         echo("")
 
 
+@legal_hold.command()
+@click.option(
+    "--matter-id", type=str, help="Filter results by legal hold UID",
+)
+@click.option(
+    "--event-type",
+    type=click.Choice(
+        [
+            "MembershipCreated",
+            "MembershipReactivated",
+            "MembershipDeactivated",
+            "HoldCreated",
+            "HoldDeactivated",
+            "HoldReactivated",
+            "Restore",
+        ]
+    ),
+    help="Filter results by event types",
+)
+@click.option("--begin", **BEGIN_DATE_DICT)
+@click.option("--end", **END_DATE_DICT)
+@format_option
+@sdk_options()
+def events(state, matter_id, event_type, begin, end, format):
+    """Report on legal hold events"""
+    formatter = OutputFormatter(format, _EVENT_KEYS_MAP)
+    events = _get_all_events(state.sdk, matter_id, begin, end, event_type)
+    if events:
+        formatter.echo_formatted_list(events)
```

Comment:
`echo_formatted_list` works well with small lists, but we should handle larger lists via a pager.

Like this:

```
        if len(events) > 10:
            output = formatter.get_formatted_output(events)
            click.echo_via_pager(output)
        else:
            formatter.echo_formatted_list(events)
```